frontend/projects/ui/src/app/app.component.ts

```diff
@@ -38,7 +38,15 @@ export class AppComponent implements OnDestroy {
     readonly themeSwitcher: ThemeSwitcherService,
   ) {}
 
-  ngOnInit() {
+  async ngOnInit() {
+    if (location.protocol === 'http:') {
+      // see if site is available securely
+      const res = await fetch(window.location.href.replace('http', 'https'))
+      if (res && res.status === 200) {
+        // redirect
+        window.location.protocol = 'https:'
+      }
+    }
```

Comment:
redirect @dr-bonez 
wont work for localhost, but tested setting `window.location.protocol = 'https:'` when on http of a current live server and it redirects properly